app/settings/profile.coffee

```diff
@@ -16,6 +16,22 @@ class Profile extends React.Component
       onClick: (e) =>
         @props.setActiveProfile(@props.profile.id)
       @textOrInput()
+      React.createElement(
+        'span'
+        className: 'profile-remove'
+        onClick: (e) =>
+          tempState = {}
```

Comment:
Set this as a function, `removeProfile`, on Profiles and pass it through the props like with `setActiveProfile`. Should be very similar to `createProfile`.

Reply:
updated to be a removeProfile function @npezza93

Reply:
updated to be a removeProfile function